client/src/Components/Mentors.js

```diff
@@ -5,59 +5,118 @@ import {
 	Route,
 	Link,
 } from "react-router-dom";
-import Questions from "../mockData/Questions.json";
+// import Questions from "../mockData/Questions.json";
 
 import "../App.css";
 import "../grid.css";
 
-export default function Mentors() {
-	const [fetchedQuestions,setfetchedQuestions]=useState(Questions);
+export default function Mentors(props) {
+	// const [fetchedQuestions,setfetchedQuestions]=useState(props.questions);
 	const [newQuizzQuestions, setNewQuizzQuestions]=useState([]);
+	const [refreshQuestions, setRefreshQuestions]=useState(false);
+
 	const [newQuizz,setNewQuizz]=useState(	{
-		id: 1,
-		name: "javaScrypt week 1",
-		publiShingDate: "2020-08-04",
-		questions: [],
+
+		name: "",
+		publiShingDate: "",
+		questions_id: [],
 	});
+
+
+
 	useEffect(() => {
-		if(newQuizz.questions!==null){
 
 
+
+		const makeQuestions=()=>{
 			let newQuizQuestions = [];
-			newQuizQuestions = newQuizz.questions.map((selId)=>{
+			newQuizQuestions = newQuizz.questions_id.map((selId)=>{
 
-				let found = (Questions.find((question)=>question.id==selId));
+				let found = (props.questions.find((question)=>question._id==selId));
 				newQuizQuestions.push(found);
-
-
 				setNewQuizzQuestions(newQuizQuestions);
-
 			}
 			);
-		}
-	},[newQuizz]);
+
+		};
+		makeQuestions();
+	},[newQuizz.questions_id]);
+
+
+
 	const addQuestion =(e)=>{
+		console.log(e.target.value);
 		setNewQuizz({ ...newQuizz,
-			questions:[...newQuizz.questions,e.target.value],
+			questions_id:[...newQuizz.questions_id,e.target.value],
 		},
 		);
 
 	};
+	const submitNewQ =()=>{
+		if(newQuizz.name.length<8){
+
+			alert("Quiz name should have at least 8 sybols");
+
+		} else if(newQuizzQuestions.length<5){
+
+			alert("Quizz  should have at least 5 questions");
+
+		} else{
+			sendQuiz();
+		}
+	};
+	const sendQuiz =()=>{
+		alert("done");
+		fetch("http://localhost:3100/api/quiz", { method:"POST",headers: { "Content-type": "application/json" },
+			body: JSON.stringify(newQuizz) })
+			.then((response) => response.json())
+			.then((data) =>console.log(data))
+			.catch((err) => console.error(err));
+
+
+
+	};
+	const newQuizName =(e)=>{
+
+		setNewQuizz({ ...newQuizz,
+			name:e.target.value,
+		},
+		);
+
+
+	};
+
 	const removeQuestion =(e)=>{
-		setNewQuizzQuestions(newQuizzQuestions.filter((obj) => obj.id != e.target.value));
+		console.log(newQuizz.questions_id);
+
+		let filterIdArr=newQuizz.questions_id.filter((boo) => {
```

Comment:
Can you rename `boo`? Presumably each one is either an `id` or a `questionId`, but `boo` doesn't mean anything.